samples/detect.v2beta1.js

```diff
@@ -0,0 +1,854 @@
+        console.log(`Knowledge base... ${r.name}`);
+      });
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END dialogflow_list_knowledge_base]
+}
+
+function deleteKnowledgeBase(projectId, knowledgeBaseFullName) {
+  // [START dialogflow_delete_knowledge_base]
+  // Instantiate a DialogFlow client.
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow KnowledgeBasesClient.
+  var client = new dialogflow.KnowledgeBasesClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  // const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+
+  client
+    .deleteKnowledgeBase({
+      name: knowledgeBaseFullName,
+    })
+    .then(responses => {
+      const result = responses[0];
+      console.log(`Name: ${result.name}`);
+      console.log(`displayName: ${result.displayName}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END dialogflow_delete_knowledge_base]
+}
+
+function createDocument(
+  projectId,
+  knowledgeBaseFullName,
+  documentPath,
+  documentName,
+  knowledgeTypes,
+  mimeType
+) {
+  // [START dialogflow_create_document]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow Documents client.
+  var client = new dialogflow.DocumentsClient({
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  // const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+  // const documentPath = `path of the document you'd like to add, e.g. https://dialogflow.com/docs/knowledge-connectors`;
+  // const documentName = `displayed name of your document in knowledge base, e.g. myDoc`;
+  // const knowledgeTypes = `The Knowledge type of the Document. e.g. FAQ`;
+  // const mimeType = `The mime_type of the Document. e.g. text/csv, text/html,text/plain, text/pdf etc.`;
+  var request = {
+    parent: knowledgeBaseFullName,
+    document: {
+      knowledgeTypes: [knowledgeTypes],
+      displayName: documentName,
+      contentUri: documentPath,
+      source: `contentUri`,
+      mimeType: mimeType,
+    },
+  };
+
+  client
+    .createDocument(request)
+    .then(data => {
+      const operation = data[0];
+      return operation.promise();
+    })
+    .then(data => {
+      console.log(`Document created`);
+      const response = data[0];
+      console.log(`Content URI...${response.contentUri}`);
+      console.log(`displayName...${response.displayName}`);
+      //console.log(`knowledgeTypes...${response.knowlegeTypes[0]}`);
```

Comment:
❗️ **#BLOCKER**

Fix this. Unsure why it's commented out. Check the canonical for what to print.

Reply:
I believe this is the canonical for this snippet: https://github.com/dialogflow/dialogflow-python-client-v2/blob/master/samples/document_management.py#L96-L103

Reply:
Please replace `...` with `: `

```js
console.log(`displayName...${response.displayName}`);

// should actually be...

console.log(`displayName: ${response.displayName}`);
```

We don't use `...`